eox_tagging/api/v1/viewset.py

```diff
@@ -95,3 +97,55 @@ def __get_user(self):
             "owner_id": {"username": user.username},
             "owner_type": "user",
         }
+
+    def create(self, request, *args, **kwargs):
+        """Creates a tag.
+
+        Args:
+            request: Django request.
+
+        Returns:
+            Django Response.
+        """
+        @audit_method(action="create-tag-api-v1")
+        def create_tag(request_data):
```

Comment:
@andrey-canon since you know the audit-model decorator better than anyone. Is this how it is supposed to work?
Should we be creating an internal method to unpack the data and be able to audit a call to an API?

Perhaps we should decorate the call to `dispatch` instead?

Reply:
if you decorate the method `create `the model with store as inputs the string representation of  self, request, args y kwargs, so this is not mandatory but this help to show  as input the data of interest

Reply:
Yes, that is correct, but the string representation of the request is of no use. Args  and Kwargs are null as well.

Since this is a viewset we don't have access to dispatch, right?

What if we alter the audit-model function to store a custom representation of a request instead of __str__? one that is better for our case?